zstacklib/zstacklib/gpu/vendors/amd.py

```
@@ -0,0 +1,111 @@
+            cmd = cmd.replace(" ", "|")
+        return cmd
+    
+    @classmethod
+    def parse_metrics(cls, output):
+        results = []
+        if not output:
+            return results
+            
+        try:
+            data = json.loads(output.strip())
+            for card_name, card_data in data.items():
```

Ruff (0.14.13)

Loop control variable `card_name` not used within loop body

Rename unused `card_name` to `_card_name`

(B007)